test/data/co2_PR_example.yaml

```diff
@@ -0,0 +1,124 @@
+  thermo:
+    model: NASA7
+    temperature-ranges: [200.0, 1000.0, 3500.0]
+    data:
+    - [4.19864056, -2.0364341e-03, 6.52040211e-06, -5.48797062e-09, 1.77197817e-12,
+      -3.02937267e+04, -0.849032208]
+    - [3.03399249, 2.17691804e-03, -1.64072518e-07, -9.7041987e-11, 1.68200992e-14,
+      -3.00042971e+04, 4.9667701]
+    note: L8/89
+  equation-of-state:
+    model: Peng-Robinson
+    a: [5.998873E+11, 0]
+    b: 18.9714
+    w_ac: 0.344
+- name: H2
+  composition: {H: 2}
+  thermo:
+    model: NASA7
+    temperature-ranges: [200.0, 1000.0, 3500.0]
+    data:
+    - [2.34433112, 7.98052075e-03, -1.9478151e-05, 2.01572094e-08, -7.37611761e-12,
+      -917.935173, 0.683010238]
+    - [3.3372792, -4.94024731e-05, 4.99456778e-07, -1.79566394e-10, 2.00255376e-14,
+      -950.158922, -3.20502331]
+    note: TPIS78
+  equation-of-state:
+    model: Peng-Robinson
+    a: [2.668423E+10, 0]
+    b: 16.5478
+    w_ac: -0.22
+- name: CO
+  composition: {C: 1, O: 1}
+  thermo:
+    model: NASA7
+    temperature-ranges: [200.0, 1000.0, 3500.0]
+    data:
+    - [3.57953347, -6.1035368e-04, 1.01681433e-06, 9.07005884e-10, -9.04424499e-13,
+      -1.4344086e+04, 3.50840928]
+    - [2.71518561, 2.06252743e-03, -9.98825771e-07, 2.30053008e-10, -2.03647716e-14,
+      -1.41518724e+04, 7.81868772]
+    note: TPIS79
+  equation-of-state:
+    model: Peng-Robinson
+    a: [1.607164E+11, 0]
+    b: 24.6549
+    w_ac: 0.049
+- name: CH4
+  composition: {C: 1, H: 4}
+  thermo:
+    model: NASA7
+    temperature-ranges: [200.0, 1000.0, 3500.0]
+    data:
+    - [5.14987613, -0.0136709788, 4.91800599e-05, -4.84743026e-08, 1.66693956e-11,
+      -1.02466476e+04, -4.64130376]
+    - [0.074851495, 0.0133909467, -5.73285809e-06, 1.22292535e-09, -1.0181523e-13,
+      -9468.34459, 18.437318]
+    note: L8/88
+  equation-of-state:
+    model: Peng-Robinson
+    a: [2.496344E+11, 0]
+    b: 26.8028
+    w_ac: 0.01
+- name: O2
+  composition: {O: 2}
+  thermo:
+    model: NASA7
+    temperature-ranges: [200.0, 1000.0, 3500.0]
+    data:
+    - [3.78245636, -2.99673416e-03, 9.84730201e-06, -9.68129509e-09, 3.24372837e-12,
+      -1063.94356, 3.65767573]
+    - [3.28253784, 1.48308754e-03, -7.57966669e-07, 2.09470555e-10, -2.16717794e-14,
+      -1088.45772, 5.45323129]
+    note: TPIS89
+  equation-of-state:
+    model: Peng-Robinson
+    a: [1.497732E+11, 0]
+    b: 19.8281
+    w_ac: 0.022
+- name: N2
+  composition: {N: 2}
+  thermo:
+    model: NASA7
+    temperature-ranges: [300.0, 1000.0, 5000.0]
+    data:
+    - [3.298677, 1.4082404e-03, -3.963222e-06, 5.641515e-09, -2.444854e-12,
+      -1020.8999, 3.950372]
+    - [2.92664, 1.4879768e-03, -5.68476e-07, 1.0097038e-10, -6.753351e-15,
+      -922.7977, 5.980528]
+    note: '121286'
+  equation-of-state:
+    model: Peng-Robinson
+    a: [1.485031E+11, 0]
+    b: 28.0810
+    w_ac: 0.04
+
+
+reactions:
+- equation: CO2 + H2 <=> CO + H2O  # Reaction 1
+  rate-constant: {A: 1.2E+3, b: 0, Ea: 0}
```

Comment:
You need a newline at the end of this file please.

Reply:
Although since you have the changes in `thermo-models.yaml` do we need this file at all?

Reply:
This file was used in PengRobinson_Test.cpp previously. Now this test has been modified to read the phase from the thermo-models.yaml. The file can be deleted, but since it is a sample yaml file containing the Peng-Robinson phase, user might want to see that as a reference.